pom.xml

```diff
@@ -192,6 +192,11 @@
                 <artifactId>icu4j</artifactId>
                 <version>77.1</version>
             </dependency>
+            <dependency>
+                <groupId>io.github.classgraph</groupId>
```

Comment:
Guava and Apache Commons Lang3 do not support scanning to list all files in the compiled classpath.


 The [classgraph](https://github.com/classgraph/classgraph) make code much simpler

Reply:
Thanks for the discovery this little lib look very helpful